miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/display/RealMiniAppDisplay.kt

```diff
@@ -45,18 +47,23 @@ internal class RealMiniAppDisplay(
 
     override fun getMiniAppView(): View = this
 
-    override fun destroyView() {
+    override fun destroyView() = clearUp()
+
+    @OnLifecycleEvent(Lifecycle.Event.ON_DESTROY)
+    fun clearUp() {
```

Comment:
The function needs to be public in order to use the `@OnLifecycleEvent` annotation? Could you just add the annotation directly to `destroyView` and avoid having two methods?

Reply:
I think merging is better as well. Updated it.
(Developer Documentation had no scopes, though works with private as well).